docs/DeveloperGuide.md

```diff
@@ -739,79 +767,79 @@ Return to [table of contents](#table-of-contents).
 
 **MSS**
 
-1. User requests to add event
-2. User specifies the details of the event
-3. JobFestGo adds the event to collection of events
-4. JobFestGo shows updated list of events
+1. User requests to add event and specifies the details of the event.
+
+2.  JobFestGo adds the event to collection of events.
+
+3.  JobFestGo shows updated list of events.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. Any of the mandatory fields not specified
+* 1a. Any of the mandatory fields not specified.
 
-    * 1a1. JobFestGo informs user that mandatory fields not specified
+    * 1a1. JobFestGo informs user that mandatory fields not specified.
 
       Use case resumes at step 1.
 
-* 1b. Event name already exists.
+* 1b. Event name already exists
 
     * 1b1. JobFestGo informs user that event name already exists.
 
       Use case resumes at step 1.
 
```

Comment:
We are not including the case when event name is not alphanumeric?

Reply:
Yup, needs to be added. Added in the extension. Thanks for the reminder!